symfit/core/minimizers.py

```diff
@@ -270,7 +271,7 @@ def execute(self, **minimizer_kwargs):
         # TODO: Compile all previous results in one, instead of just the
         # number of function evaluations. But there's some code down the
         # line that expects scalars.
-        final.infodict['nfev'] = sum(ans.infodict['nfev'] for ans in answers)
```

Comment:
It would be nice if Objectives kept track of the number of (function|jacobian|hessian) evaluations. Different PR though.

Reply:
I agree that would be nice, but indeed it should be a different PR. The important thing is that now at least we catch all results under `minimizer_output`, so at least it's there if you really need it.